lbrynet/core/Wallet.py

```diff
@@ -749,7 +759,8 @@ def _decode_and_cache_claim_result(self, claim, update_caches):
                                                         decoded, claim['address'],
                                                         claim['height'],
                                                         claim['amount'], claim['supports'],
-                                                        claim.get('channel_name', None),
+                                                        claim.get(
+                                                            'channel_name', None),
                                                         claim.get('signature_is_valid', None))
```

Comment:
Needs to be formatted better, move arguments to new line below the function definition and indented 4 times.